ftl/common/constants.py

```diff
@@ -27,3 +27,16 @@
 PACKAGE_JSON = 'package.json'
 NODE_DEFAULT_ENTRYPOINT = 'node server.js'
 NPMRC = '.npmrc'
+
+P1_CACHE_STR = '{key_version}:{language}->{key}'
```

Comment:
What are P1 and P2 supposed to mean?

Reply:
Sorry these names are terrible. This is for phase 1 and phase 2, because in phase 1 we don't have individual package information. I'll make these a little more clear